Header header

bool abort
bool reset